### DISCUSSION TITLE
New Terminology for GitHub Previews

### DISCUSSION THREAD

**Discussion by spaltrowitz** [community/community#141918] [Announcements]
As part of our commitment to improving your experience at GitHub, we’re simplifying the terminology we use to refer to products that are in testing and validation stages. Starting on **October 18, 2024**, you’ll start seeing the word “Preview” instead of “Alpha” or “Beta” to describe our features that are not yet generally available. 

**What’s Changing?**

Our goal with this update is to create a more consistent, clear process that helps our customers understand the state of new features and how they fit into their development workflows.

* As shown in the table below, we’re reducing the number of terms we’re using  but keeping the same flexibility for giving early access and gathering customer feedback before a General Availability (GA) launch.
* The key difference between “Private” and “Public” previews is whether the release is publicly announced. 

**What to Expect**

These changes are now live in customer-facing documentation as of **today**.

**Here’s an overview of the changes:**

<p align="center">
<img width="637" alt="Screenshot 2024-10-18 at 1 05 23 PM" src="https://github.com/user-attachments/assets/55165ef0-27a8-464c-b878-8bd1fec282c1">
 

Thanks for being part of the GitHub community! These updates are designed to provide clearer communication and a smoother experience as we roll out new features.

**Comment by Mayaleeeee** [comment]
Thanks for the announcement, @spaltrowitz 

I’m eager to contribute to the GitHub community. Are there any open issues related to design, research, documentation, or community management where I could get involved? Additionally, are there any individuals you could refer me to for potential research collaboration?

I would love to be an active part of the community wherever possible.

My name is Maya, and I am the Lead Design Maintainer at AsyncAPI, an Open Source community. I also serve on the Technical Steering Committee and the Code of Conduct Committee.

**Reply by chrisblanch39** [reply]
>

**Comment by chrisblanch39** [comment]
### _
<details><summary>Details</summary>
<p>



</p>
</details> >_ _`[https://github.com/orgs/community/discussions/141918#discussion-7338515](url)`_

**Comment by misterob0t** [comment]
Thanks for the announcement, @spaltrowitz!

Are there any open issues in these areas where I could help? I'm also interested in potential research collaborations if you could connect me with relevant individuals.
I remain available to actively contribute wherever my skills would be valuable to the community.

Best regards,
Adil Aziz.

**Comment by JorenVsyntra** [comment]
Hey everyone!

**Comment by JorenVsyntra** [comment]
everything good
